### PR TITLE
Update to python 3.9 (1956)

### PR DESCRIPTION
**Pull request by estyxx**
#1956

### CODE REVIEW

File: .github/workflows/backend-checks.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
       - uses: actions/checkout@master
       - uses: actions/setup-python@v2
         with:
-          python-version: "3.7.9"
+          python-version: "3.9.0"
```
Comment (marcoacierno) [inline .github/workflows/backend-checks.yml:17]: can you update to 3.9.1?

**Review by marcoacierno — COMMENTED** [review]
Overall looks good

Can you also update the python version in pre-commit (here: https://github.com/pythonitalia/pycon/blob/master/.pre-commit-config.yaml#L20)?

File: .github/workflows/backend-checks.yml (hunk 1 above)
Comment (estyxx) [reply .github/workflows/backend-checks.yml]: done

File: backend/Dockerfile.lambda [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ARG FUNCTION_DIR="/home/app/"
 
-FROM python:3.7-alpine as build-stage
+FROM python:3.9-alpine as build-stage
```
Comment (marcoacierno) [inline backend/Dockerfile.lambda:3]: we have another `FROM python:3.7-alpine` below

also need to change

```
COPY --from=build-stage /usr/local/lib/python3.7/site-packages /usr/local/lib/python3.7/site-packages
```

**Review by marcoacierno — APPROVED** [review]
Thanks!

**Pull request merged**